src/glossary.md

```diff
@@ -25,6 +26,7 @@ provider                |  the function that executes a query ([see more](query.
 query                   |  perhaps some sub-computation during compilation ([see more](query.html))
 sess                    |  the compiler session, which stores global data used throughout compilation
 side tables             |  because the AST and HIR are immutable once created, we often carry extra information about them in the form of hashtables, indexed by the id of a particular node.
+soundness               |  soundness is a technical term in type theory. Roughly, if a type system is sound, then if a program type-checks, it is type-safe; i.e. I can never (in safe rust) force a value into a variable of the wrong type. (see "completeness").
```

Comment:
typo: I can never ...

Reply:
Sorry, what is the typo here?

Reply:
Oops sorry, I wrongly read this sentence.